arrayfire/array.py

```diff
@@ -1185,20 +1185,35 @@ def to_list(self, row_major=False):
         ct_array, shape = self.to_ctype(row_major, True)
         return _ctype_to_lists(ct_array, len(shape) - 1, shape)
 
-    def __repr__(self):
+    def to_string(self):
```

Comment:
I've checked VS Code and it's not affected, it doesn't call `str` internally. I'll do as you suggested, but I have to find a way to detect PyCharm. Will be back with the changes.